src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -137,42 +137,47 @@ export function CollapsibleNav({
 
       <EuiHorizontalRule margin="none" />
 
-      <EuiFlexItem className="eui-yScroll">
-        {/* Recently viewed */}
-        <EuiCollapsibleNavGroup
-          key="recentlyViewed"
-          title={i18n.translate('core.ui.recentlyViewed', { defaultMessage: 'Recently viewed' })}
-          isCollapsible={true}
-          initialIsOpen={getIsCategoryOpen('recentlyViewed', storage)}
-          onToggle={isCategoryOpen => setIsCategoryOpen('recentlyViewed', isCategoryOpen, storage)}
-        >
-          {recentNavLinks.length > 0 ? (
-            <EuiListGroup
-              aria-label={i18n.translate('core.ui.recentlyViewedAriaLabel', {
-                defaultMessage: 'Recently viewed links',
-              })}
-              listItems={recentNavLinks.map(link => {
-                // TODO #64541
-                // Can remove icon from recent links completely
-                const { iconType, ...linkWithoutIcon } = link;
-                return linkWithoutIcon;
+      {/* Recently viewed */}
+      <EuiCollapsibleNavGroup
+        key="recentlyViewed"
+        background="light"
+        title={i18n.translate('core.ui.recentlyViewed', { defaultMessage: 'Recently viewed' })}
+        isCollapsible={true}
+        initialIsOpen={getIsCategoryOpen('recentlyViewed', storage)}
+        onToggle={isCategoryOpen => setIsCategoryOpen('recentlyViewed', isCategoryOpen, storage)}
+      >
+        {recentNavLinks.length > 0 ? (
+          <EuiListGroup
+            aria-label={i18n.translate('core.ui.recentlyViewedAriaLabel', {
+              defaultMessage: 'Recently viewed links',
+            })}
+            listItems={recentNavLinks.map(link => {
+              // TODO #64541
+              // Can remove icon from recent links completely
+              const { iconType, ...linkWithoutIcon } = link;
+              return linkWithoutIcon;
+            })}
+            maxWidth="none"
+            color="subdued"
+            gutterSize="none"
+            size="s"
+            className="eui-yScroll"
+            style={{ maxHeight: '9.5em', marginRight: '-8px' }}
```

Comment:
Can you now move this to a custom style?
```suggestion
            className="kbnCollapsibleNav__recentsListGroup"
```

```scss
. kbnCollapsibleNav__recentsListGroup {
  @include euiYScroll;
  max-height: $euiSize * 8;
  margin-right: -$euiSizeS;
}